### PR TITLE
quincy: rgw/qa: Add test suite for lifecycle cases

### PR DESCRIPTION
**Pull request by soumyakoduri**
Execute lifecycle s3-tests in the teuthology test-suite by configuring
required storage classes and 'rgw lc debug interval' option.

Backport of https://github.com/ceph/ceph/pull/44939
Fixes: https://tracker.ceph.com/issues/54387
Signed-off-by: Soumya Koduri <skoduri@redhat.com>

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [x] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by soumyakoduri** [comment]
retest make check

**Comment by soumyakoduri** [comment]
jenkins test make check

**Comment by soumyakoduri** [comment]
https://pulpito.ceph.com/soumyakoduri-2022-02-27_10:07:34-rgw-wip-skoduri-quincy-distro-basic-smithi/

File: qa/suites/rgw/lifecycle/tasks/rgw_s3tests.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+tasks:
+- install:
+- ceph:
+- rgw: [client.0]
+- s3tests:
+    client.0:
+      force-branch: ceph-master
```
Comment (cbodley) [inline qa/suites/rgw/lifecycle/tasks/rgw_s3tests.yaml:7]: can you please update this to `force-branch: ceph-quincy` to match https://github.com/ceph/ceph/pull/44867?

Comment (soumyakoduri) [reply qa/suites/rgw/lifecycle/tasks/rgw_s3tests.yaml:7]: done

Comment (cbodley) [reply qa/suites/rgw/lifecycle/tasks/rgw_s3tests.yaml:7]: can you please push your update? i don't see it here yet

Comment (soumyakoduri) [reply qa/suites/rgw/lifecycle/tasks/rgw_s3tests.yaml]: oops..done

**Review by cbodley — APPROVED** [review]

**Pull request merged**